docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,421 @@ aliases:
 
 # Brigade Gateways
 
-This guide explains how gateways work, and provides guidance for creating your own
-gateway.
+This guide explains how gateways work, and provides guidance for creating your
+own.
 
 ## What Is A Brigade Gateway?
 
-The [Brigade architecture](design.md) is oriented around the concept that Brigade
-scripts run as a response to one or more events. In Brigade, a _gateway_ is an
-entity that generates events. Often times, it translates some external trigger
-into a Brigade event.
+The [Brigade architecture](/topics/design) is oriented around the concept that
+Brigade scripts run as a response to one or more events. In Brigade, a gateway
+is an entity that generates events. Often times, it translates some external
+trigger into a Brigade event.
 
-Brigade ships with the ability to enable various gateways that are ready to go.
+While there is no default gateway that ships with Brigade, installing one
```

Comment:
I'm slightly worried that "default gateway" (singular) and "installing one" leave this sentence open to misinterpretation-- that gateways exist in a 1:1 with a Brigade instance.